Command/CleanUpCommand.php

```diff
@@ -45,6 +45,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $this->cleanUpExpiredJobs($em, $con, $input);
         $this->collectStaleJobs($em);
+
+        return 0;
```

Comment:
Please use `\Symfony\Component\Console\Command\Command::SUCCESS` const

Reply:
OK, now it uses the const